src/pages/en/blog/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 lang_de: false
-page-title: Current blog about the Corona-Warn-App
-page-description: Stay up-to-date with blog posts on open-source project Corona-Warn-App. The Corona-Warn-App is an app that helps trace infection chains of COVID-19 in Germany.
+page-title: Open-Source Project Corona-Warn-App - Blog
+page-description: Blog for the Corona-Warn-App. Here you get current information about the developmen on the Corona-Warn-App and all important updates.
```

Comment:
Typo
"developmen on" =>
"development of"

Reply:
Corrected in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/2ab687f10cc2cc171e62681496d7121a087c8d5a

Reply:
you changed to development, but you didn't change "on" to "of".

https://dictionary.cambridge.org/dictionary/english/development gives an example of "development" followed by "of"

Reply:
@MikeMcC399 You're right, changed in https://github.com/corona-warn-app/cwa-website/pull/1332/commits/74ad7df09b7525bc6cf4d827ffc319b91ba2deec.